video/out/opengl/common.c

```diff
@@ -34,15 +34,15 @@
 #include "utils.h"
 
 // This guesses if the current GL context is a suspected software renderer.
+// Note we don't attempt to blacklist Microsoft's fallback implementation.
+// It only provides OpenGL 1.1 and will be skipped anyway.
```

Comment:
I'd rather put it inside the function just before the `return`

Reply:
Done